frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-local-volume-set-step/create-local-volume-set-step.tsx

```diff
@@ -108,100 +150,120 @@ export const CreateLocalVolumeSet: React.FC<CreateLocalVolumeSetProps> = ({
-            setErrorMessage={setError}
-            storageClassName={storageClass.name}
-          />
-          {state.chartNodes.size < MINIMUM_NODES && (
-            <Alert
-              className="co-alert ceph-ocs-install__wizard-alert"
-              variant="danger"
-              title={t('ceph-storage-plugin~Minimum Node Requirement')}
-              isInline
-            >
-              {t(
-                'ceph-storage-plugin~A minimum of 3 nodes are required for the initial deployment. Only {{nodes}} node match to the selected filters. Please adjust the filters to include more nodes.',
-                { nodes: state.chartNodes.size },
-              )}
-            </Alert>
-          )}
-          <RequestErrors errorMessage={error?.message} inProgress={inProgress} />
-        </>
-      )}
+    <ErrorHandler
+      loaded={discoveriesLoaded}
+      loadingMessage={
+        !csvLoaded
+          ? t('ceph-storage-plugin~Checking LSO operator installation')
+          : !discoveriesLoaded
+          ? t('ceph-storage-plugin~Discovering disks on all hosts. This may take a few minutes.')
+          : null
+      }
+      error={discoveriesLoadError}
+      errorMessage={csvLoadError || csv?.status?.phase !== 'Succeeded' ? <LSOInstallAlert /> : null}
+    >
+      <>
+        <Grid>
+          <GridItem lg={8} md={8} sm={8}>
+            <Form noValidate={false} className="odf-create-lvs__form">
+              <LocalVolumeSetBody
+                state={state}
+                dispatch={dispatch}
+                storageClassName={storageClass.name}
+                allNodes={allNodes.current}
+                nodes={nodes}
+              />
+            </Form>
+          </GridItem>
+          <GridItem
+            lg={3}
+            lgOffset={9}
+            md={3}
+            mdOffset={9}
+            sm={3}
+            smOffset={9}
+            className="odf-create-lvs__donut-chart"
+          >
+            <SelectedCapacity
+              dispatch={dispatch}
+              state={state}
+              ns={ns}
+              nodes={nodes}
+              lvdResults={lvdResults}
+            />
+          </GridItem>
+        </Grid>
+        <ConfirmationModal
+          ns={ns}
+          nodes={nodes}
+          state={state}
+          dispatch={dispatch}
+          setInProgress={setInProgress}
+          setErrorMessage={setError}
+          storageClassName={storageClass.name}
+          stepIdReached={stepIdReached}
+        />
+        {state.chartNodes.size < MINIMUM_NODES && (
+          <Alert
+            className="odf-create-lvs__alert"
+            variant="danger"
+            title={t('ceph-storage-plugin~Minimum Node Requirement')}
+            isInline
+          >
+            {t(
+              'ceph-storage-plugin~A minimum of 3 nodes are required for the initial deployment. Only {{nodes}} node match to the selected filters. Please adjust the filters to include more nodes.',
```

Comment:
```suggestion
              'ceph-storage-plugin~A minimum of 3 nodes are required for the initial deployment. Only {{nodes}} node match to the selected filters. Please adjust the filters to include more nodes.',
```
`node` needs to be pluralized based on the value of `nodes`.